service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -100,22 +100,24 @@ private void run1466(CaseDetails caseDetails) {
             migrationId, UUID.fromString("b8da3a48-441f-4210-a21c-7008d256aa32")));
     }
 
-    private void run1501(CaseDetails caseDetails) {
-        var migrationId = "DFPL-1501";
-        var possibleCaseIds = List.of(1659711594032934L);
-
+    private void run1612(CaseDetails caseDetails) {
+        var migrationId = "DFPL-1612";
+        var possibleCaseIds = List.of(1687780363265112L);
+        UUID documentId = UUID.fromString("db163749-7c8a-45fe-88dd-63641560a9d9");
+        CaseData caseData = getCaseData(caseDetails);
         migrateCaseService.doCaseIdCheckList(caseDetails.getId(), possibleCaseIds, migrationId);
-        caseDetails.getData().putAll(migrateCaseService.removeFurtherEvidenceSolicitorDocuments(
-            getCaseData(caseDetails), migrationId, UUID.fromString("43a9287c-f840-4104-958f-cbd98d28aea3")));
+        migrateCaseService.verifyUrgentDirectionsOrderExists(caseData, migrationId, documentId);
+        caseDetails.getData().put("urgentDirectionsOrder", "");
```

Comment:
I think if we're removing the field completely we can do 
```suggestion
        caseDetails.getData().remove("urgentDirectionsOrder");
```
(may wanna double check that one though!)